src/screens/AccountNetworkChooser.tsx

```diff
@@ -126,8 +127,25 @@ function AccountNetworkChooser({
 		</ScrollView>
 	);
 
-	const sortNetworkKeys = ([, networkParams]: [any, NetworkParams]): number =>
-		networkParams.protocol !== NetworkProtocols.SUBSTRATE ? 1 : -1;
+	const sortNetworkKeys = (
+		[, params1]: [any, NetworkParams],
+		[, params2]: [any, NetworkParams]
+	): number => {
+		const infinity = 999;
+		const getOrder = (param: NetworkParams): number => {
+			if (isEthereumNetworkParams(param)) return infinity;
+			if (isUnknownNetworkParams(param)) return 0;
+			return (param as SubstrateNetworkParams).order ?? infinity - 1;
+		};
```

Comment:
This seems fragile.

Reply:
Delete this part, now give each network a `order` field in order to make an explicit sorting.

In the future it would be nice to enable user to drag network in the list and re-order them. This could be implement after we have custom network feature ready with #524.